HW04/Anastasiya_Gavrilik/spec/custom/my_each_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe Custom do
+  describe 'my_each' do
+    let(:arr) { [1, 2, 3] }
+
+    before { Array.include(described_class) }
+
+    context 'when empty block given' do
+      subject { arr.my_each }
+
+      it 'returns self' do
+        expect(subject).to eq arr
+      end
```

Comment:
We can use one line syntax 😉
```suggestion
      it { is_expected.to eq arr }
```

Reply:
thanks for your  advice